packages/search-ui-app-search-connector/package.json

```diff
@@ -43,6 +42,7 @@
     "typescript": "^4.5.4"
   },
   "dependencies": {
+    "@elastic/search-ui": "^1.9.0",
```

Comment:
Moved search-ui core package from devDependencies to dependencies, as I need to import a helper from there.